lib/common/graph.js

```diff
@@ -220,7 +220,10 @@ class ConcertoGraphVisitor extends DiagramVisitor {
                 const fqn = modelFile.getFullyQualifiedTypeName(argument.name);
                 if (fqn) {
                     parameters.graph.addVertex(fqn);
-                    (parent === modelFile) ? parameters.graph.addEdge(parent.getNamespace(), fqn) : parameters.graph.addEdge(parent.getFullyQualifiedName(), fqn);
+                    (parent === modelFile) ? parameters.graph.addEdge(parent.getNamespace(), fqn)
+                        : (parent instanceof ClassDeclaration)
```

Comment:
Can you drop the nested ternary operator and improved readability (and maintainability) ?